client/js/shout.js

```diff
@@ -382,6 +382,28 @@ $(function() {
 		}
 	}
 
+	if ($.cookie("timestamp")) {
+		$("#timestamp").val($.cookie("timestamp")).change();
+	} else {
+		$("#timestamp").val("HH:mm").change();
+	}
+
+	$("#timestamp").on("change", function() {
+		var timestamp = $(this).val();
+		Handlebars.unregisterHelper("tz");
+		Handlebars.registerHelper(
+			"tz", function(time) {
+				if (time) {
+					var utc = moment.utc(time, timestamp).toDate();
+					return moment(utc).format(timestamp);
+				} else {
+					return "";
+				}
+			}
+		);
+		$.cookie("timestamp", timestamp);
```

Comment:
Yeah, it's alright, I'm changing how this is done anyway, I'm modifying the generic cookie setting.